lib/Controller/UsersController.php

```diff
@@ -362,29 +372,32 @@ public function create($username, $password, array $groups= [], $email='') {
 		$currentUser = $this->userSession->getUser();
 
 		if (!$this->isAdmin) {
-			if (!empty($groups)) {
-				foreach ($groups as $key => $group) {
+			// Is probably subadmin
+			// FIXME: Can not admin actually create users ??
```

Comment:
IMO this is building the list of groups that a subadmin has "control" over. In the webUI user management page, a subadmin only sees the groups that they are subadmin of. So they can only adjust membership of those.

Also, they cannot remove a user from all groups that the subadmin has "control" of. i.e. a subadmin cannot "lose control" of a user in their group(s). But a subadmin who is subadmin of multiple groups can add/move users between those groups.

The UI enforces that OK (before this PR - I will test this PR soon to see if there is any regression). We should check the private API used underneath here, to make sure a subadmin cannot "hack" add extra groups in their browser console and then get any level of control over those groups.